### PR TITLE
Feat skip button on unit page  - LESQ-1048

### PR DESCRIPTION
**Pull request by JBR90**
## Description

Music year: 2011

- Adds skip button to threads on unit page

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2769--oak-web-application.netlify.thenational.academy
Got to unit and specialist unit page - tab through page, you should see a skip to units button that will skip thread filters

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Comment by RBrownsett** [comment]
@JBR90 just checking I'm not sure if the focus is working correctly - the ticket mentions that selecting the button will move focus to the top of list of units, but when I select the button it seems to move focus to around half way down the page. Example video below:


https://github.com/user-attachments/assets/b25f87a1-113c-463b-8480-c58f3706a26b

File: src/components/TeacherComponents/UnitsLearningThemeFilters/UnitsLearningThemeFilters.tsx [hunk 1]
```diff
@@ -70,29 +78,45 @@ const UnitsLearningThemeFilters = ({
...
-              ...linkProps,
-              search: { ...linkProps.search, ["learning-theme"]: undefined },
-            },
-          },
-          ...learningThemesMapped.map(({ label, slug }) => ({
-            label: label ? label : "",
-            linkProps: {
-              ...linkProps,
-              search: { ...linkProps.search, ["learning-theme"]: slug },
+    <OakThemeProvider theme={oakDefaultTheme}>
```
Comment (njblore) [inline src/components/TeacherComponents/UnitsLearningThemeFilters/UnitsLearningThemeFilters.tsx:81]: 👀 should this be here?

Comment (JBR90) [reply src/components/TeacherComponents/UnitsLearningThemeFilters/UnitsLearningThemeFilters.tsx:81]: No should be in the tests instead

**Comment by JBR90** [comment]
> @JBR90 just checking I'm not sure if the focus is working correctly - the ticket mentions that selecting the button will move focus to the top of list of units, but when I select the button it seems to move focus to around half way down the page. Example video below:
> 
>  Screen.Recording.2024-09-11.at.16.58.09.mov

Hey @RBrownsett , It's working for we with tabbing onto the button and pressing enter ? - which matches the behaviour of the skip buttons on the home page and search

**Review by RBrownsett — APPROVED** [review]
Approved - tested, focus is working correctly - does not show in mobile (which is expected)

**Pull request merged**